synapse/handlers/auth.py

```diff
@@ -297,10 +304,11 @@ def login_with_password(self, user_id, password):
         defer.returnValue((user_id, access_token, refresh_token))
 
     @defer.inlineCallbacks
-    def login_with_cas_user_id(self, user_id):
+    def login_with_user_id(self, user_id):
```

Comment:
Perhaps:

def get_access_token_for_user_id(self, user_id):
    """
    Gets an access token for the user with the given user ID.
    The user is assumed to have been authenticated by some other mechanism (e.g. CAS).
    """